<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.wipro.practice.SeleniumPractice.OpenCartTest" tests="2" time="0.000" ignored="2" failures="0" timestamp="01 Feb 2018 09:52:24 GMT" skipped="1" hostname="D-113064624" errors="1">
  <testcase name="beforeMethod" time="0.000" classname="com.wipro.practice.SeleniumPractice.OpenCartTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.wipro.practice.SeleniumPractice.ExtentReportBaseClass.beforeMethod(ExtentReportBaseClass.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:169)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- beforeMethod -->
  <testcase name="loginToOpenCartTest" time="0.000" classname="com.wipro.practice.SeleniumPractice.OpenCartTest">
    <skipped/>
  </testcase> <!-- loginToOpenCartTest -->
  <testcase name="deinitializeExtentReport" classname="com.wipro.practice.SeleniumPractice.ExtentReportBaseClass">
    <skipped/>
  </testcase> <!-- deinitializeExtentReport -->
  <testcase name="initializeExtentReport" classname="com.wipro.practice.SeleniumPractice.ExtentReportBaseClass">
    <skipped/>
  </testcase> <!-- initializeExtentReport -->
</testsuite> <!-- com.wipro.practice.SeleniumPractice.OpenCartTest -->
